src/timezone.ts

```diff
@@ -0,0 +1,39 @@
+import { Message } from 'discord.js';
+import * as moment from 'moment-timezone';
+
+import config from './config';
+
+const { defaultEmbed, TIMEZONES } = config;
+
+export const timezone = async (message: Message) => {
+    if (message.author.bot) {
+        return;
+    }
+
+    const match = message.content.match(/([\d]{1,2})([:\d]{3})?[\s]?(pm|am|AM|PM)?\b[\s]([a-zA-Z]{3})?\b/);
+
+    if (match) {
+        const mentioned = match[0];
+        const hour = match[1];
+        const minutes = match[2];
+        const dayNight = match[3];
+        const zone = match[4];
```

Comment:
maybe we could short this out to 1 line?
```suggestion
        const [mentioned, hour, minutes, dayNight, zone] = match;
```

Reply:
Oh I really like this 👍